modules/nf-core/proovframe/fix/main.nf

```diff
@@ -0,0 +1,49 @@
+process PROOVFRAME_FIX {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/proovframe:0.9.7--hdfd78af_1':
+        'biocontainers/proovframe:0.9.7--hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(fa)
+    tuple val(meta2), path(tsv)
+
+    output:
+    tuple val(meta), path("*.fa"), emit: out_fa
+    path "versions.yml"          , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    proovframe   \\
+        fix \\
+        ${args} \\
+        -o ${prefix}.fa  \\
+        ${fa} \\
+        ${tsv}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        proovframe: 0.9.7
```

Comment:
This one should be extracted from the tool directly, something like: https://github.com/nf-core/modules/blob/cb08035150685b11d890d90c9534d4f16869eaec/modules/nf-core/fastqc/main.nf#L49

Reply:
We tried really hard to make that happen..but really couldn't figure out a way..

Reply:
Ok think I found a way, brb :P

Reply:
Changed it for the proovframe/map module as well